catalogue/webapp/utils/dates.ts

```diff
@@ -162,3 +162,35 @@ export function extendEndDate(params: {
     }
   }
 }
+
+export function isValidDate(params: {
```

Comment:
Can we rename this to something that better describes what it does?  Seems like it's (a) not operating on just 1 date and (b) is Wellcome Collection-specific in that it considers closed days etc

Reply:
Yeah, it's checking whether the selected date falls within a range and is also not a day on which the library is closed.
Not sure how to capture that in a name? Does isRequestableDate work?